spec/features/users/user_profile_spec.rb

```diff
@@ -62,6 +64,36 @@
         expect(current_path).to eq account_path
         expect(page).to have_content(t('idv.messages.personal_key'))
       end
+
+      it 'allows the user reactivate their profile by reverifying' do
+        profile = create(:profile, :active, pii: { ssn: '1234', dob: '1920-01-01' })
+        user = profile.user
+
+        sign_in_live_with_2fa(user)
+
+        profile.deactivate(:password_reset)
+        profile.reload
+        visit account_path
+
+        click_on t('account.index.reactivation.reverify')
+        click_idv_begin
+
+        # not using the helper method as the generic password doesn't seem to work
+        fill_out_idv_form_ok
+        click_idv_continue
+        fill_out_financial_form_ok
+        click_idv_continue
+        click_idv_address_choose_phone
+        fill_out_phone_form_ok(user.phone)
+        click_idv_continue
+        fill_in 'Password', with: user.password
```

Comment:
You know, I was actually testing in the browser this by manually setting profiles to require a password reset, I think it probably makes more sense to run through the real reset UI 😄